### PR TITLE
MONIT 32317: Add a tag(format) to hyperlogs metrics

### PR DESCRIPTION
**Pull request by deosu**
As part of this PR, I have
1. added a tag(format=<format_value>) to the following metrics
    .logs.[port].received.batches
    .logs.[port].received.total
    .logs.[port].received.bytes
    .logs.[port].received.tagLength
    .logs.[port].received.tagCount
    .logs.[port].received.messageLength
    .logs.[port].received.max-burst-rate
    .logs.[port].received.lag
    .logs.[port].sent
    .logs.[port].blocked
    .logs.[port].rejected
    .logs.[port].discarded

2. added the following new metrics for cloud logs ingestion
    .logs.[port].logs_json_cloudwatch.received
    .logs.[port].logs_json_cloudwatch.received.max-burst-rate

3. fixed a couple of bugs(because of which following metrics were not incremented)
    .logs.[port].received.total
    .logs.[port].sent

4. refactored some code

### CODE REVIEW

**Review by xuranchen — APPROVED** [review]

File: proxy/pom.xml [hunk 1]
```diff
@@ -386,7 +386,7 @@
     <dependency>
       <groupId>com.wavefront</groupId>
       <artifactId>java-lib</artifactId>
-      <version>2023-01.1</version>
+      <version>2023-01.31-alpha-3</version>
```
Comment (joannak-vmware) [inline proxy/pom.xml:389]: Waiting for java-lib release

**Review by xuranchen — APPROVED** [review]

File: proxy/pom.xml (hunk 1 above)
Comment (deosu) [reply proxy/pom.xml]: changed

**Review by joannak-vmware — APPROVED** [review]
LGTM

**Review by laullon — APPROVED** [review]
LGTM

**Pull request merged**